src/pages/editor/PublishStrategySettings.tsx

```diff
@@ -71,74 +93,207 @@ export const PublishStrategySettings = () => {
+          </Box>
+
+          <Divider sx={{ my: '2.5rem' }} />
+
+          <Typography variant="h2">{t('editor.retentions_strategy.institution_rrs')}</Typography>
+
+          <Formik
+            initialValues={customer}
+            onSubmit={async (values) => await updateRightsRetentionStrategy.mutateAsync(values)}>
+            {({ values, isSubmitting, setFieldValue }: FormikProps<CustomerInstitution>) => {
+              const isRrs =
+                values.rightsRetentionStrategy?.type === RightsRetentionStrategyTypes.RightsRetentionStrategy;
+              const isOverridableRrs =
+                values.rightsRetentionStrategy?.type ===
+                RightsRetentionStrategyTypes.OverridableRightsRetentionStrategy;
+              const isNullRrs =
+                values.rightsRetentionStrategy?.type === RightsRetentionStrategyTypes.NullRightsRetentionStrategy;
+
+              return (
+                <Box component={Form} sx={{ maxWidth: '35rem' }}>
+                  <Field name={'rightsRetentionStrategy.type'}>
+                    {({ field }: FieldProps<RightsRetentionStrategyTypes>) => (
+                      <FormControlLabel
+                        label={t('editor.retentions_strategy.rights_retentions_strategy')}
+                        control={
+                          <Checkbox
+                            data-testid={dataTestId.editor.rightsRetentionsStrategy}
+                            {...field}
+                            disabled={isSubmitting}
+                            checked={isRrs || isOverridableRrs}
+                            value={values.rightsRetentionStrategy?.type}
+                            onChange={() => {
+                              setFieldValue(
+                                field.name,
+                                isRrs || isOverridableRrs
+                                  ? RightsRetentionStrategyTypes.NullRightsRetentionStrategy
+                                  : RightsRetentionStrategyTypes.RightsRetentionStrategy
+                              );
+                              setFieldValue('rightsRetentionStrategy.id', '');
+                            }}
+                          />
+                        }
+                      />
+                    )}
+                  </Field>
+
+                  <Field name="rightsRetentionStrategy.id">
+                    {({ field }: FieldProps<string>) => (
+                      <FormLabel
+                        component="legend"
+                        sx={{ color: 'primary.main', fontWeight: 'bold', marginTop: '1rem' }}>
+                        {t('editor.retentions_strategy.institution_rrs_info_page')}
+
+                        <TextField
+                          type="url"
+                          data-testid={dataTestId.editor.rightsRetentionsStrategyLink}
+                          {...field}
+                          label={t('editor.retentions_strategy.institution_rrs_url')}
```

Comment:
Lurer på om `label` er overflødig her, siden `<FormLabel>` rett over gjør samme jobben?
```suggestion
```

Reply:
Alternativt er det å flytte denne nøkkelen til `commons`, og gi bruke store bokstaver for teksten der: "URL"
```suggestion
                          label={t('common.url')}
```